src/components/TravelInfo.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from "react";
+import { TextField, InputLabel } from "@mui/material";
+import style from "./TravelInfo.module.css";
+
+function TravelInfo(props: any) {
+  const [businessTime, setBusinessTime] = useState("");
```

Comment:
Other option here would be to have one state hook for the whole form, example
`const [form, setForm] = useState({})`

Then we would have only one handler:
```
// TODO: typing key variants 'departuredTime' | 'businessDate' etc.
function handleForm(e: any, key: string) {
    setForm({
    ...form,
    [key]: e.target.value 
    });
  }
```

And you would call it like this:
`onChange={e => handleForm(e,'businessTime')}`